.gitignore

```diff
@@ -25,3 +25,5 @@ erl_crash.dump
 
 node_modules/
 .DS_Store
+test/support/minio_data/.minio.sys/*
```

Comment:
i think you can safely delete these lines now that we've changed the way that we pre-load data into the minio container image during testing.